packages/renderer/src/lib/deployments/DeploymentDetailsSummary.svelte

```diff
@@ -1,11 +1,10 @@
 <script lang="ts">
 import type { V1Deployment } from '@kubernetes/client-node';
 import ErrorMessage from '../ui/ErrorMessage.svelte';
-import DeploymentColumnConditions from './DeploymentColumnConditions.svelte';
-import type { DeploymentUI } from './DeploymentUI';
-import DeploymentColumnPods from './DeploymentColumnPods.svelte';
+import ObjectMeta from '../kube/details/ObjectMeta.svelte';
+import Deployment from '../kube/details/Deployment.svelte';
```

Comment:
please add prefix /suffix with Kubernetes and details so with the className you see in one sec if it's a generic, specific, etc class